wen-restart/src/wen_restart.rs

```diff
@@ -558,6 +559,24 @@ pub(crate) fn aggregate_restart_heaviest_fork(
     } else {
         info!("Not enough stake agreeing with our heaviest fork: slot: {}, bankhash: {}, stake aggreeing with us {} out of {}",
             heaviest_fork_slot, heaviest_fork_hash, total_active_stake_agreed_with_me, total_active_stake);
+        let mut max_slot_hash = (0, Hash::default());
+        let mut max_stake = 0;
+        for (slot, hash) in block_stake_map.keys() {
+            let stake = block_stake_map[&(*slot, *hash)];
+            if stake > max_stake {
+                max_stake = stake;
+                max_slot_hash = (*slot, *hash);
+            }
+            info!(
+                "Slot: {}, Hash: {}, Stake: {}",
+                slot,
+                hash,
+                block_stake_map[&(*slot, *hash)]
+            );
+        }
+        warn!("Max stake slot: {}, hash: {}, stake: {}% does not agree with my choice, please go to discord 
+               to download the snapshot and restart the validator with --wait-for-supermajority.",
```

Comment:
nit: We should only point them to discord to download a snapshot if the `max stake` > `wait_for_supermajority_threshold_percent as f64 - 5.0`

I think we should have a separate message:

`Cluster consensus slot: {}, hash: {}, stake: {}% does not agree, please go to discord...`

Reply:
Done